manifest-build-tools/application/update_changelog.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+# Copyright 2015-2016, EMC, Inc.
+
+"""
+The script update changelog under debian
+
+usage:
+./on-tools/manifest-build-tools/HWIMO-BUILD on-tools/manifest-build-tools/application/update_changelog.py \
+--build-dir d/ \
+--version 1.2.6 \
+--publish \
+--git-credential https://github.com,GITHUB \
+--message "new branch 1.2.6"
+
+The required parameters: 
+build-dir: The top directory which stores all the cloned repositories
+version: The new release version
+git-credential: url, credentials pair for the access to github repos.
+
+The optional parameters:
+message (default value is "new release" + version )
+publish: If true, the updated changlog will be push to github.
+git-credential: url, credentials pair for the access to github repos.
+                If publish is true, the parameter is required.
+"""
+import os
+import sys
+import argparse
+import datetime
+import subprocess
+from RepositoryOperator import RepoOperator
+from common import *
+
+class ChangelogUpdater(object):
+    def __init__(self, repo_dir, version):
+        """
+        The module updates debian/changelog under the directory of repository
+        __repo_dir: the directory of the repository
+        __version: the new version which is going to be updated to changelog
+        """
+        self._repo_dir = repo_dir
+        self._version = version
+        self.repo_operator = RepoOperator()
+ 
+    def debian_exist(self):
+        """
+        check whether debian or debianstatic directory under the repository
+        return: True if debian or debianstatic exist
+                False
+        """
+        if os.path.isdir(self._repo_dir):
+            for filename in os.listdir(self._repo_dir):
+                if filename == "debian":
```

Comment:
The description of this function says "debian or debianstatic", but here you only check `debian`, please keep them aligned.